ts/nni_manager/common/manager.ts

```diff
@@ -108,6 +108,9 @@ abstract class Manager {
 
     public abstract getTrialJobStatistics(): Promise<TrialJobStatistics[]>;
     public abstract getStatus(): NNIManagerStatus;
+
+    public abstract getTrialOutputLocalPath(trialJobId: string): Promise<string>;
```

Comment:
Who should be calling these two functions?

Reply:
tensorboard manager will call these.

Reply:
Is it necessary to use two functions?
I suppose `subpath` of `fetchTrialOutput` is always the return value of `getTrialOutputLocalPath`?

Reply:
`getTrialOutputLocalPath` just returns the root local path where will or already have the trial output data, then we can start the tensorboard immediately, we won't wait for downloading data finished or even don't care if we need to download data. 
And `subpath` means the relative path of the return value of `getTrialOutputLocalPath`, like the local path is `local/trial_id/output`, the remote path is `remote/trial_id/output` , and subpath is `tensorboard`, then I hope to download the file under the `remote/trial_id/output/tensorboard` to the `local/trial_id/output/tensorboard`.